src/registrar/assets/sass/_theme/_usa-modal.scss

```diff
@@ -0,0 +1,7 @@
+@use "cisa_colors" as *;
+@use "uswds-core" as *;
+
+.usa-modal.wider-modal {
```

Comment:
Did you try the USWDS large modal `usa-modal--lg`? 

I think that could work. Related - if you ever actually need to extend a base CSS class with a variant, I like the -- convention (usa-modal--wider)

Reply:
Looking at this again, the content should work fine in a default modal. We're using those for similarity sized content

Reply:
usa-modal--lg doesn't match the figma in terms of content-height and some other padding things. The reasoning for the implementation here is that most domain names tend to wrap their content, making it harder to read 
(stuff like this-plea-tea.gov becomes this-
plea-tea.gov). I think this should be kept for consistency with the design, but I am open to discussion on it

That said, I like the `usa-modal--wider` naming convention quite a lot